subgraph/src/unlock.ts

```diff
@@ -87,7 +87,7 @@ export function handleNewLock(event: NewLock): void {
 
   // maxKeysPerAddress set to 1 prior to lock v10
   lock.maxKeysPerAddress = BigInt.fromI32(1)
-  if (version.ge(BigInt.fromI32(9))) {
+  if (version.ge(BigInt.fromI32(10))) {
```

Comment:
Hum. That does not seem right. `ge` is greater or equal, right? If we we should keep it to 9!

Reply:
yes it was a mistake in the previous subgraph code :/